### PR TITLE
Enh/1/learning

### LINKED ISSUE #1: Issue
- SSH access to Tiger Github
- Create private repo name mle-training
- Create Github issue 
- Create new branch and copy code snippet 
- Add .gitignore file to repo and avoid unnecessary files like **pyc** files.
- Set up a python development environment named **mle-dev** to run the script and export the environments file as env.yml 
- Create a Pull Request (PR) as per expected standards
- Add the facilitators as collaborators

### PR DESCRIPTION
**Pull request by irish98**

### CODE REVIEW

**Review by RoshanRoy97 — CHANGES_REQUESTED** [review]
1. Remove redundant zone identifier files
2. Add required contents to Readme.md, plus instructions on how to activate environment and run the code, with markdown code highlight
4. Screenshot  of code run attached to PR
5. Attach issue to PR

**Comment by RoshanRoy97** [comment]
Please attach a screenshot of the output from running the nonstandardcode.py successfully in the mle-dev environment as a part of your PR description.
Just click on the bottom part of the comment box where it says 'Attach files by dragging ...'

**Review by RoshanRoy97 — COMMENTED** [review]
The readme is to be copy pasted from the link provided in Assignment 2, exercise 4.
Just format the code snippets with proper markdown syntax.

**Comment by irish98** [comment]
ss1

File: env.yml [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+  - toml=0.10.2=pyhd3eb1b0_0
+  - tornado=6.2=py310h5eee18b_0
+  - tzdata=2022g=h04d1e81_0
+  - wheel=0.37.1=pyhd3eb1b0_0
+  - xz=5.2.8=h5eee18b_0
+  - zlib=1.2.13=h5eee18b_0
+  - zstd=1.5.2=ha4553b6_0
+  - pip:
+      - pyqt5-sip==12.11.0
+prefix: /home/rishabh/ENTER/envs/mle-dev
```
Comment (RoshanRoy97) [inline env.yml]: Remove the prefix line. This is because we want the environment to be replicated anywhere.

**Comment by RoshanRoy97** [comment]
Remember to request or rerequest a review when needed. This is in addition to adding the reviewer on a PR. There's an option to request review if you point near the reviewer id.

**Comment by RoshanRoy97** [comment]
Please update the readme with previous instructions you had as well. Copy pasted from assignment link + updates you wrote on how to run the code.
Quoting the assignment:
4. Create a new branch (enh/<issue-id>/<short-msg>) and copy the [code-snippet here](https://drive.google.com/drive/folders/1ebqWUF06mkLRpmV-AjAjPMYgmh0cuUvX) and commit as-is including the README. Use appropriate commit messages as per the guidelines.
7.Update the README file with instructions on how to run the code.

File: env.yml (hunk 1 above)
Comment (irish98) [reply env.yml]: ok

**Review by RoshanRoy97 — APPROVED** [review]
Moving forward, attach screenshots, if any, to the PR itself.
More importantly, please ensure the that the changes here carry forward to your following assignments. For example, changes on 1.2 are reflected on 1.3, and so on, if applicable.
So, please update accordingly.

Approved

**Comment by RoshanRoy97** [comment]
Remember to merge after approval.

**Comment by irish98** [comment]
> please ensure the that the changes here carry forward to

sure, Thanks Roshan for your feedback.

**Comment by RoshanRoy97** [comment]
Please see that you're following all instructions in the assignment carefully:

10.Once the PR is approved, merge the PR to master branch and [give it a tag ](https://git-scm.com/book/en/v2/Git-Basics-Tagging)(named version) `v0.1` and push that to the remote

**Comment by irish98** [comment]
> Please see that you're following all instructions in the assignment carefully:
> 
> 10.Once the PR is approved, merge the PR to master branch and [give it a tag ](https://git-scm.com/book/en/v2/Git-Basics-Tagging)(named version) `v0.1` and push that to the remote

It's done. I have raised the PR for Assignment 1.3, thank you.

**Pull request merged**